Build/Strategic/Strategic_Movement.cc

```diff
@@ -3178,13 +3178,6 @@ BOOLEAN GroupWillMoveThroughSector( GROUP *pGroup, UINT8 ubSectorX, UINT8 ubSect
 			{
 				dy = -1;
 			}
-			else
-			{
-				Assert( 0 );
-				pGroup->ubSectorX = ubOrigX;
-				pGroup->ubSectorY = ubOrigY;
-				return TRUE;
-			}
```

Comment:
Yeah, I thought it was somehow related and was trying to prevent ugly merge conflicts...